app/actors/sufia/create_with_files_actor.rb

```diff
@@ -0,0 +1,42 @@
+module Sufia
+  # Creates a work and attaches files to the work
+  class CreateWithFilesActor
+    attr_reader :work_actor, :uploaded_file_ids
+
+    def initialize(work_actor, uploaded_file_ids)
+      @work_actor = work_actor
+      @uploaded_file_ids = uploaded_file_ids
+    end
+
+    delegate :visibility_changed?, to: :work_actor
+
+    def create
+      validate_files && work_actor.create && attach_files
```

Comment:
Updated `validate_files`, but `attach_files` only returns true, it can't return anything else.